code/modules/research/designs/medical_designs.dm

```diff
@@ -25,11 +25,11 @@
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
 
 /datum/design/bluespacebeaker
-	name = "Bluespace Beaker"
+	name = "Pre-War Polymer Beaker"
```

Comment:
Should probably be called a saturnite beaker. Also, the description still says bluespace!

Reply:
Should be fixed. I was about here in my pass of design recipes when I realized that all the designs would have to be gone through, and their names and descriptions updated to better fit the fallout theme.